cli/kata-check.go

```diff
@@ -7,26 +7,49 @@
 
 package main
 
+import (
+	"unsafe"
+)
+
 /*
 #include <linux/kvm.h>
+#include <stdint.h>
 
 const int ioctl_KVM_CREATE_VM = KVM_CREATE_VM;
+
+//TODO; Have IOCTL request until ACRN patches are upstreamed.
```

Comment:
would you mind raising an issue and mentioning it here?, that way we can track it

Reply:
Removed TODO and raised https://github.com/kata-containers/runtime/issues/1784.